clicker-react-app/src/components/Clicker.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+
+class Clicker extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      count: 0
+    }
+  }
+
+  counterMinus = () => {
+    this.setState({
+      count: this.state.count - 1 
```

Comment:
It's a common mistake!..
We shouldn't base on `this.state.count` , it could be changed in another time due to react optimizations. Instead, you should use a function notation: `this.setState((state) => ({ count: state.count - 1 }))`